tools.js

```diff
@@ -130,20 +173,19 @@ module.exports = (function () {
 			});
 		}
 	}
+
 	Tools.loadMods = function () {
```

Comment:
This function appears to be a ton of try blocks in a row. I think it's consolidated enough not to need the functions split out...